### PR TITLE
Aleksei/indent fix

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

Css style fixes for browser extension. 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/common/TmSessionImport.vue [hunk 1]
```diff
@@ -1,87 +1,87 @@
...
-      if (this.$v.seed.$invalid || this.$v.seed.$invalid) return
-      this.$router.push("/recover/name")
-    }
-  },
-  validations: () => ({
-    seed: { required, words24 }
-  })
-}
-</script>
+<template>
```
Comment (faboweb) [inline src/components/common/TmSessionImport.vue]: something is wrong here. why is this showing every line as changed?

**Comment by faboweb** [comment]
unit test failing and changelog missing

**Comment by jbibla** [comment]
@iambeone something is not right here. there are some linting conflicts - i just a saw PR yesterday switching `"` to `'`...

can you investigate?

also, share a screenshot of the fix if it is stlying related.

**Comment by iambeone** [comment]
@jbibla, i have updated this branch to exclude any conflicts. 

image
image

**Comment by Bitcoinera** [comment]
@iambeone you had only one easy block left, "ci/circleci: testUnit". You can go around it easily by updating your snapshots with the `yarn test:unit -u` command. In general, I always execute this command every time I make a PR to the FE just to be sure.

I also checked with `yarn lint --fix` and there are no lint errors left...

**Comment by Bitcoinera** [comment]
> this is not how i would solve this issue. i would add media queries around the style that is causing this so the styles are only used on mobile - not in the extension or on the browser.
> 
> also, this totally breaks the desktop view:

I think this is fixed in [here](https://github.com/luniehq/lunie/pull/3439). Mmm... Does this PR include something that is not present in #3439 @iambeone ?

I think either we close this one or take out the CSS changes from #3439 .

**Comment by iambeone** [comment]
Yes, it's is already fixed in a another PR

**Comment by Bitcoinera** [comment]
> Yes, it's is already fixed in a another PR

OK, then I think it is better if we close this one

**Pull request closed without merging**